fpm/src/fpm.F90

```diff
@@ -28,9 +49,9 @@ logical function exists(filename) result(r)
 subroutine cmd_build()
 logical :: src
 print *, "# Building project"
-src = exists("src/fpm.f90")
+src = exists("src/fpm.F90")
 if (src) then
-    call run("gfortran -c src/fpm.f90 -o fpm.o")
+    call run("gfortran -c src/fpm.F90 -o fpm.o")
 end if
 call run("gfortran -c app/main.f90 -o main.o")
```

Comment:
I suggest that we tell the compiler explicitly to preprocess using `-cpp` flag (`-fpp` on ifort, not sure about others), rather than relying on the implicit rule of .F90 vs .f90.

So, keep sources as .f90 and preprocess explicitly with `-cpp`.

Reply:
I agree, but haskell fpm needs to be updated also for this to work.

Reply:
I use `.f90` now.